java/src/jmri/jmrix/AbstractSerialPortController.java

```diff
@@ -212,6 +212,15 @@ public String getCurrentPortName() {
         return mPort;
     }
 
+    private static String getSymLink(File file) {
```

Comment:
One suggestion: Call this function "getSymlinkTarget(File symlink)" or something similar to make it more self-documenting.

Reply:
Thanks. Done.